QEfficient/cloud/infer.py

```diff
@@ -230,6 +206,14 @@ def main(
         default=-1,
         help="Effort level to reduce the on-chip memory",
     )
+    #FIXME: Add verbose feature
+    parser.add_argument(
+        "--verbose","-v",
+        action="store_true",
```

Comment:
set to `true`, default?

Reply:
We can, @anujgupt-github What do you say?

Reply:
No, I mean this should be optional, we should set to `false `by default and enable when user is interested to get more details via verbose.

Reply:
Yes, that how `store_true` works

Reply:
Got it. closing